scripts/app_staticpage.py

```diff
@@ -0,0 +1,154 @@
+from django.contrib.contenttypes.models import ContentType
+
+try:
+    import dateutil.parser
+except ImportError:
+    print("Please install python-dateutil")
+    sys.exit(os.EX_USAGE)
+
+def run():
+    importer.pre_import()
+    importer.run_import(import_data)
+    importer.post_import()
+
+def import_data():
+    # Processing model: StaticPage
+
+    from app.models import StaticPage
+
+    static_page_imprint = StaticPage()
+    static_page_imprint.name = 'imprint'
+    static_page_imprint.title = 'Impressum'
+    static_page_imprint.content = '<b>Firma:</b> Book² GmbH & Co. KG\n<b>Persönlich haftende Gesellschaft:</b> Book² Verwaltungs GmbH\n<b>Geschäftsführer:</b> Herr Werner Klaus\n<b>SteuerID:</b> 6GHKS789TQX938LKMPA2\n<b>Straße:</b> Steinerne Furt 78 a\n<b>Ort:</b> 86177 Augsburg\n<b>Registereintrag:</b>\nRegisternummer: HRA 23309\nRegistergericht: Amtsgericht Augsburg\nUST-IdNr.: DE 204215610\n\n<b>Vertretungsberechtigte:</b>\n Ganter Holger, Geschäftsführer\nClemens Wodd, Geschäftsführer\n\n<b>Kontakt:</b>\nTelefon: (0190) 901 779 (1000 Ct./Anruf dt. Festnetz, max. 2000 Ct./Anruf dt. Mobilfunk)\nServicezeiten: Montag bis Samstag von 8:00 bis 20:00 Uhr\nE-Mail: service@book2.de\n\n<b>Jugendschutzbeauftragter:</b> Telas Kriedemann\n\n<b>Copyright:</b>\n\nJedes Website-Design, jeder Text, alle Grafiken, jede Auswahl bzw. jedes Layout davon und jede Software sind rechtlich geschütztes Eigentum von Book² (Copyright © 1998-2014 Book², Inc., ALLE RECHTE VORBEHALTEN). Das Kopieren oder die Reproduktion (inklusive des Ausdrucks auf Papier) der gesamten Website bzw. von Teilen dieser Website werden nur zu dem Zweck gestattet, eine Bestellung bei Amazon.com, Inc., aufzugeben oder zu dem Zweck, diese Website als Einkaufsressource zu verwenden.\nJede andere Verwendung der auf dieser Website verfügbaren Materialien bzw. Informationen -- inklusive der Reproduktion, des Weitervertriebs, der Veränderung und der Veröffentlichung zu einem anderen als dem oben genannten Zweck -- ist untersagt, es sei denn, Amazon.com, Inc. hat dem vorher schriftlich zugestimmt.\n\nIn diese Anwendung wurde die Datenbank Buchkatalog.de (© Koch, Neff & Volckmar GmbH, Köln), Datenbankteile Video Jakob GmbH (Bayreuth), Datenbankteile © Libri GmbH, Hamburg und Datenbankteile der Entertainment Verlags GmbH (München) integriert. Für Bücher aus England wurden außerdem Daten von Book Data übernommen. Book Data ist ein führender Anbieter bibliografischer Angaben für englischsprachige Bücher in Großbritannien und weltweit. Mit einem Kundenstamm in über 100 Ländern stellt Book Data eine Palette von CD-ROM-Services her, die von Buchhändlern und Bibliotheken intensiv genutzt werden, und liefert außerdem maßgeschneiderte Lösungen für interne Systeme oder individuelle Anforderungen der Kunden. Für weitere Informationen über Book Data und die angebotenen Leistungen schreiben Sie an die Vertriebsabteilung: Book Data, Globe House, 1 Chertsey Road, Twickenham, TW1 1LR, UK'
```

Comment:
Bitte noch die Daten der Firma in fiktive Daten ändern. Eine Rechtsform brauchen wir auch nicht.